src/components/children/NestedInfoTable.jsx

```diff
@@ -52,8 +50,8 @@ class NestedInfoTable extends React.Component {
 														return (<div key={hazardDataset.datasetId}>
 															<ListItem key={hazardDataset.datasetId}>
 																<Button color="primary" variant="contained"
-																	size="small" className={classes.inlineButtons}
-																	onClick={() => this.props.onClick(hazardDataset.datasetId)}>Preview</Button>
+																		size="small" className={classes.inlineButtons}
```

Comment:
Can you fix the eslint errors? I see a few for this file "error Expected indentation of 17 tabs but found 18  indent"

Reply:
Reformated